Modules/Core/Support/MpdfHelper.php

```
@@ -0,0 +1,33 @@
+    bool $stream = true,
+    $password = null,
+    $isInvoice = null,
+    $is_guest = null,
+    bool $embed_xml = false,
+    ?array $associated_files = []
+)
+    {
+        // Implementation moved from procedural function
+        // TODO: Copy implementation from original helper file
+        throw new \RuntimeException('Method not yet implemented');
+    }
```

PHPMD (2.15.0)

The method pdf_create is not named in camelCase. (undefined)

(CamelCaseMethodName)